### ISSUE TITLE
Clicking on buttons doesn't make a tap sound if the "sound on tap" option is set.

### ISSUE THREAD

**Issue by sethladd** [flutter/flutter#12438]
Report from Hacker News comments. They observed that Flutter Gallery seems to be missing this behavior.

**Comment by eseidelGoogle** [comment]
My understanding is we added this behavior on request for the Hamilton app.  Maybe we failed to wire it up for the Gallery as well?

**Comment by sethladd** [comment]
If it "just works" work, then probably all we need to do is republish the gallery.

**Comment by sethladd** [comment]
Cool, after turning on "sound on tap" and rebooting my phone (I kid you not) I tested in the latest (unpublished) Gallery.  I hear "sound on tap" now! Closing this. We'll need to republish the Gallery.
